apollo-router/src/services/execution/service.rs

```diff
@@ -252,6 +252,16 @@ impl ExecutionService {
         nullified_paths: &mut Vec<Path>,
         mut response: Response,
     ) -> Option<Response> {
+        /*
+        println!("\nin process_graphql_response");
+        println!("query: ${:?}", query);
+        println!("operation_name: ${:?}", operation_name);
+        println!("variables: ${:?}", variables);
+        println!("is_deferred: ${:?}", is_deferred);
+        println!("response: ${:?}", response);
+        println!("nullified_paths: ${:?}", nullified_paths);
+        */
```

Comment:
to delete

Reply:
Sorry, I should have re-read my PR but i was making changes late last night. I'll clear these up.